about.html

```diff
@@ -5,10 +5,123 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>IT.LIVE iraq 2019 | About</title>
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/about.css" />
 </head>
 
 <body>
+    <header class="header column">
+        <div class="header__nav flex bg-dark">
+            <a href="#"><img src="images/icons/facebook.svg" alt="facebook" /></a>
+            <a href="#"><img src="images/icons/twitter.svg" alt="twitter" /></a>
+            <div class="button-group flex">
+                <button type="button" class="btn btn-text">English</button>
+                <button type="button" class="btn btn-text">My Page</button>
+                <button type="button" class="btn btn-text">Logout</button>
+            </div>
+        </div>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
```

Comment:
- [ ] Kindly fix the closing button it doesn't work

https://user-images.githubusercontent.com/55596970/158561315-4f83d24f-a0c6-4a9a-8efb-41d83e5249a6.mp4

Reply:
Adressed 👍🏻